lib/perl/Genome/Model/SingleSampleGenotype/Command/HaplotypeCaller/BucketIterator.pm

```diff
@@ -56,6 +56,11 @@ sub _run {
             $self->fatal_message('Could not %s for interval %s.', $mode, $interval);
         }
 
+        unless($ENV{UR_DBI_NO_COMMIT}) {
+            #save our progress after creating a new result
```

Comment:
It's not strictly necessary.  I included it to avoid repeatedly syncing databases and then rolling back as it iterated.